js/ui/scheduler/workspaces/ui.scheduler.work_space.js

```diff
@@ -1223,7 +1223,7 @@ class SchedulerWorkSpace extends WidgetObserver {
 
         if(this.isVirtualScrolling()) {
             const virtualScrollingState = this.virtualScrollingDispatcher.getState();
-            Object.assign(options, {
+            extend(options, {
```

Comment:
@AntonSermyazhko can we use extend here?

Reply:
Same here.